index.js

```diff
@@ -0,0 +1,132 @@
+const inquirer = require("inquirer");
+const Manager = require("./src/Manager");
+const Engineer = require("./src/Engineer");
+const Intern = require("./src/Intern");
+const generateHTML = require("./src/generateHTML");
+const fs = require("fs");
+
+const team = [];
+
+const createAnotherEmployeeQuestion = {
+  type: "list",
+  message: "Select option:",
+  name: "option",
+  choices: ["Engineer", "Intern", "Done"],
+};
+
+const managerQuestions = [
+  {
+    type: "input",
+    message: "Name of the manager?",
+    name: "name",
+  },
+  {
+    type: "input",
+    message: "Employees ID?",
+    name: "id",
+  },
+  {
+    type: "input",
+    message: "Email addres?",
+    name: "email",
+  },
+  {
+    type: "input",
+    message: "Office number?",
+    name: "officeNumber",
+  },
+  createAnotherEmployeeQuestion,
+];
+
+const engineerQuestions = [
```

Comment:
nice job listing out all of your vars/constants at the top of this file - fab for readability! :-)